requirements.txt

```diff
@@ -1,6 +1,6 @@
 urllib3>=1.8.3
 elasticsearch>=2.4.0,<3.0.0
 click>=6.0
-pyyaml>=3.10
```

Comment:
You may need to rebase against master. This set of requirements is wrong. _Very_ outdated. This may be a part of what's causing your errors in CI. Look at the current [requirements.txt](https://github.com/elastic/curator/blob/master/requirements.txt) from the master branch to see what I mean.

Reply:
No, this is a PR for the 4.3 branch, which has these dependencies. Yes, it is intentional, we need this in the 4.3 branch.

Reply:
Then you might as well maintain your own branch, then. I'm no longer publishing releases to the 4.x branch of Curator, as the last one of those was released the 10th of April 2017.

Why are you pinned on such an old branch? Do you really have that old a version of Curator you're trying to support? The 5.x releases support Elasticsearch 5.x - 7.x, so I am imagining Elasticsearch 2.x here, which is also EOL and no longer supported by Elastic.

Reply:
Ok, not surprising. Yes, I know it's ancient :)